main_test.go

```diff
@@ -50,6 +60,9 @@ var (
 		Logger:                &testServiceLogger,
 		RunDatabaseMigrations: false,
 	}
+
+	redisHostPort = os.Getenv("REDIS_HOST_PORT")
```

Comment:
like for other variables used by end to end tests, these should have seperate versions of these for the test to use - makes it easier to isolate and test against redis backend for debugging production issues easier

ask: please add versions of these variables prefixed with `TEST_` and set in .env

Reply:
Implemented here: https://github.com/Kava-Labs/kava-proxy-service/pull/39/commits/8416e195552903ca5993214e698c103f0912011c

Hopefully I understood it correctly